src/routes/+layout.svelte

```diff
@@ -179,18 +179,18 @@
   ];
 
   let contentEl = $state<HTMLDivElement>();
-  let drawerLeft = $state<QDrawer>();
-  let drawerRight = $state<QDrawer>();
+  let drawerLeftEl = $state<QDrawer>();
+  let drawerRightEl = $state<QDrawer>();
```

Comment:
Not sure we need to wrap those in `$state`

Reply:
Me neither, it was like this before. Since there are already some smaller issues, I'd leave it as is for now and refactor/optimize later, once everything is resolved.